frontend/src/components/FAQHelp/HelpSearchBar.tsx

```diff
@@ -1,14 +1,12 @@
 import React, { ReactElement } from 'react';
+import { ReactComponent as SearchIcon } from '../../assets/search.svg'
```

Comment:
You can just do `import SearchIcon from  '../../assets/search.svg'` here. No need to change this though